### PR TITLE
Rework the docs to deal with pre release versions better

### PR DESCRIPTION
**Pull request by Cadair**
This is part of a change where we are only going to render changelogs to the main changelog file for **final** releases only.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -262,15 +256,11 @@ def rstjinja(app, docname, source):
 # JSOC email os env
 os.environ["JSOC_EMAIL"] = "jsoc@cadair.com"
 
-# JSOC email os env
```
Comment (nabobalis) [inline docs/conf.py:265]: How was this here twice?

File: sunpy/util/sphinx/changelog.py [hunk 2]
```diff
@@ -44,6 +44,11 @@ def run(self):
         return []
 
 
+class DummyChangelog(ChangeLog):
```
Comment (nabobalis) [inline sunpy/util/sphinx/changelog.py:47]: I like it.

**Review by nabobalis — APPROVED** [review]

File: docs/conf.py (hunk 1 above)
Comment (wtbarnes) [reply docs/conf.py:265]: That may have been my fault. I added it in the cutout PR and it was also in the metadata PR. Maybe the rebase got screwed up somehow.

**Review by wtbarnes — APPROVED** [review]

File: docs/conf.py (hunk 1 above)
Comment (nabobalis) [reply docs/conf.py:265]: Ah, all I did was merge in master so that would explain it.

**Comment by kakirastern** [comment]
🤣

**Comment by Cadair** [comment]
Both the doc fails seem unrelated.

**Comment by kakirastern** [comment]
🎉

**Pull request merged**